rule/range-loop-var.go

```diff
@@ -0,0 +1,111 @@
+package rule
```

Comment:
Please lint the code with `revive` using the recommended configuration (see README.MD) and remove all warnings/errors